modules/text2vec-contextionary/doc.go

```diff
@@ -0,0 +1,17 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2020 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+// modcontextionary concentrates some of the code that relates to the
+// contextionary module, this must be extracted when Weaviate becomes modular.
+//
+// For now it's just used to concentrate any new contextionary specific code,
+// so it's already in one place to make the modularization easier later on
+package modcontextionary
```

Comment:
this file is intentionally empty?

Reply:
It's the package doc. I think that's a pattern that you have a `doc.go` where the only content is a package-level comment?

Reply:
ok